src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/localize/templatestrings.en.json

```diff
@@ -0,0 +1,9 @@
+{
+  "author": "Microsoft",
+  "name": "API Controller",
+  "description": "An API Controller with or without read/write actions",
```

Comment:
We can remove the "an" at the beginning. `API Controller with or without read/write actions`

Reply:
sounds good ty